processor/k8sprocessor/kube/client_test.go

```diff
@@ -224,8 +224,8 @@ func TestPodDelete(t *testing.T) {
 	deleteRequest := c.deleteQueue[0]
 	assert.Equal(t, deleteRequest.ip, "1.1.1.1")
 	assert.Equal(t, deleteRequest.name, "podB")
-	assert.True(t, deleteRequest.ts.After(tsBeforeDelete))
-	assert.True(t, deleteRequest.ts.Before(time.Now()))
+	assert.True(t, !deleteRequest.ts.Before(tsBeforeDelete))
+	assert.True(t, !deleteRequest.ts.After(time.Now()))
```

Comment:
Pretty sure this was failing because Windows has less granularity of time than Linux by default, so changed this to "AfterOrEqual" ...

Reply:
Updated